pytorch_lightning/utilities/exceptions.py

```diff
@@ -15,3 +15,7 @@
 
 class MisconfigurationException(Exception):
     pass
+
+
+class DeadlockDetectedException(Exception):
+    pass
```

Comment:
rather instead of pass add short docstring with explanation...

Reply:
Yes.